Undo https://github.com/rust-lang/rust/commit/1532fd8cd0db93f469e414f9da31ef083a44fcba
[target.'x86_64-apple-darwin'] is not recognized in is not recognized in config.toml

--- src/bootstrap/configure.py.orig	2022-11-02 07:36:24.000000000 -0700
+++ src/bootstrap/configure.py	2022-12-17 12:04:19.000000000 -0700
@@ -405,7 +405,7 @@
         configured_targets.append(target)
 for target in configured_targets:
     targets[target] = sections['target'][:]
-    targets[target][0] = targets[target][0].replace("x86_64-unknown-linux-gnu", "'{}'".format(target))
+    targets[target][0] = targets[target][0].replace("x86_64-unknown-linux-gnu", target)
 
 
 def is_number(value):
